envoy_build_config/BUILD

```diff
@@ -22,5 +22,6 @@ envoy_cc_library(
         "@envoy//source/extensions/filters/network/http_connection_manager:config",
         "@envoy//source/extensions/stat_sinks/metrics_service:config",
         "@envoy//source/extensions/transport_sockets/tls:config",
+        "@envoy_mobile//library/common/extensions/filters/http/platform_extension:config",
```

Comment:
nit: the name extension suffix is a little odd, because the extension is a filter, not a generic "PlatformExtension".

Reply:
Ok, after reading more of the PR I think I see your thought here. The `PlatformExtension` namespace contains types that bridge to the platform. In this case it is the bridge filter. Usually the directory and the namespace share the filter's name, but I see how here is different.

Reply:
I'm not married to the naming, but yeah, that was my thinking at the time.

Reply:
See what you think of the updated naming.